pyproject.toml

```diff
@@ -49,7 +49,7 @@ markers = "fuzzing: Run Hypothesis fuzz test suite"
 line_length = 100
 force_grid_wrap = 0
 include_trailing_comma = true
-known_third_party = ["hypothesis"]
+known_third_party = ["ape", "click", "conftest", "eip712", "eth_account", "eth_typing", "eth_utils", "hexbytes", "hid", "pytest", "rlp", "setuptools"]
```

Comment:
Weird that it is putting `conftest` here, but that should probably be in the `known_first_party` list since it refers to a local file